src/Controls/tests/SourceGen.Tests/Controls.SourceGen.UnitTests.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AssemblyName>Microsoft.Maui.Controls.SourceGen.UnitTests</AssemblyName>
+    <IsPackable>false</IsPackable>
+    <EnableDefaultXamlItems>False</EnableDefaultXamlItems>
+    <EnableDefaultCssItems>False</EnableDefaultCssItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+   <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+		<PackageReference Include="xunit" Version="2.4.1" />
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.11.0" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.Build.Framework" Version="16.11.0" PrivateAssets="all" />
+        <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.11.0" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Core\Controls.Core-net6.csproj" GeneratePathProperty="true" PrivateAssets="all" />
+    <ProjectReference Include="..\..\src\Xaml\Controls.Xaml-net6.csproj" GeneratePathProperty="true" PrivateAssets="all" />
+    <ProjectReference Include="..\..\src\SourceGen\Controls.SourceGen.csproj" />
+    <ProjectReference Include="..\..\..\TestUtils\src\TestUtils.SourceGen\TestUtils.SourceGen.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="TestData\SimpleTest.xaml">
+      <Generator></Generator>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
IDE happy times?

Reply:
Maybe just the blank generator attribute, but the point is to have this file available to test so copying it to the expected location seems fine?